common/notify_stream/src/lib.rs

```diff
@@ -191,6 +191,9 @@ impl Watcher {
                     EventKind::Modify(ModifyKind::Data(DataChange::Any)) => {
                         Some(Event::Write(received.paths.first().unwrap().clone()))
                     }
+                    EventKind::Modify(ModifyKind::Data(DataChange::Content)) => {
+                        Some(Event::Write(received.paths.first().unwrap().clone()))
```

Comment:
did we miss it from the beginning?

Reply:
no - as far as I can tell it's a mac thing.